web/styles/search.css

```diff
@@ -2,150 +2,232 @@
+    .typeahead.dropdown-menu {
+        /* Match the typeahead's width to its parent container. */
+        right: 0;
+        top: 0;
+        margin-top: 0;
+        padding-top: 0;
+        box-shadow: none;
+        position: relative;
+        width: 100%;
+        background: var(--color-background-search);
+        color: var(--color-text-search);
+
+        .active > a {
+            background-color: var(--color-background-search-option-hover);
+            background-image: none;
+            color: var(--color-text-search-hover);
         }
+        border-width: 0;
     }
 
-    #search_query {
+    .input-append {
+        position: relative;
         width: 100%;
-        font-size: 16px;
-        height: calc(var(--header-height) - 1px);
-        min-height: 30px;
+        margin-bottom: 0;
+    }
+
+    .search-input {
+        font-size: 14px;
+        /* override bootstrap padding for input[type="text"] */
         padding: 0;
-        padding-right: 40px;
         border: none;
+        outline: none;
         border-radius: 0;
         font-family: "Source Sans 3 VF", sans-serif;
-        font-weight: 300;
-        line-height: var(--header-height);
+        font-weight: 400;
+        height: 28px;
+        /* override bootstrap min-height for input[type="text"] */
+        min-height: 28px;
+        background: transparent;
         text-overflow: ellipsis;
         overflow: hidden;
         white-space: nowrap;
-        padding-left: 35px;
-        color: hsl(0deg 0% 33%);
-
-        &:focus {
-            outline: 0;
-        }
-
-        @media (width < $sm_min) {
-            vertical-align: text-bottom;
-        }
+        flex-grow: 1;
     }
 
-    #searchbox-input-container:focus {
-        box-shadow: inset 0 0 0 2px hsl(204deg 20% 74%);
+    #searchbox-input-container {
+        display: flex;
+        align-items: center;
+        /* override .input-append style from app_components.js */
+        letter-spacing: normal;
+        height: 28px;
     }
 
-    .search_close_button {
-        position: absolute;
-        right: 0;
-        top: 6px;
-        background: none;
-        border-radius: 0;
-        border: none;
-        height: 30px;
-        text-align: center;
-        padding: 4px;
-        color: inherit;
-        opacity: 0.5;
-        font-size: 18px;
-        box-shadow: none;
-        text-shadow: none;
-        z-index: 5;
+    @media (width < $md_min) {
+        .navbar-search:not(.expanded) {
+            background: transparent;
+        }
 
-        &:hover {
+        .navbar-search:not(.expanded) .search_icon:hover {
             opacity: 1;
         }
+    }
 
-        &:disabled {
-            visibility: hidden;
+    @media (width < $sm_min) {
+        #searchbox_form {
+            margin: 0;
         }
 
-        @media (width < $sm_min) {
-            top: 0;
+        #searchbox-input-container,
+        .search-input {
+            height: var(--header-height);
```

Comment:
Oooh, here's a height with a variable on `.search-input`. Perhaps that value needs to be adjusted within the relevant media queries, if that's changing?

Reply:
Oooo yeah cool, updated to this:

```css
    /*
    Height of the search box, which appears in the header.
    On very small screens, the search box's height becomes
    the height of its parent (i.e. the header height).
    */
    --search-box-height: 28px;
    @media (width < $sm_min) {
        --search-box-height: var(--header-height);
    }
```